### PR TITLE
Enable multi role for users in admin

### PR DESCRIPTION
**Pull request by kryswisnaskas**
**Description of change**
Adds an ability to assign multiple roles to users in the admin UI.

**How to test**
- pull changes
- assign multiple roles to couple of users in the admin UI
- create reports
- verify that the roles are displayed on the landing page

**Issue(s)**
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-69

**Checklist**

- [x] Meets issue criteria
- [x] Code tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified

### CODE REVIEW

**Review by jasalisbury — APPROVED** [review]
:rocket:

**Review by dcmcand — APPROVED** [review]
Looks good!

File: frontend/src/pages/Admin/UserInfo.js [hunk 1]
```diff
@@ -1,19 +1,22 @@
+/* eslint-disable react/jsx-props-no-spreading */
 import React from 'react';
 import PropTypes from 'prop-types';
 import {
   Label, TextInput, Grid, Fieldset,
 } from '@trussworks/react-uswds';
 import moment from 'moment';
-
 import RegionDropdown from '../../components/RegionDropdown';
-import JobTitleDropdown from '../../components/JobTitleDropdown';
```
Comment (rahearn) [inline frontend/src/pages/Admin/UserInfo.js:9]: Can we delete this component? It appears to be unused now.

**Review by rahearn — CHANGES_REQUESTED** [review]
Looks good, works well. For code cleanup, is there any reason to keep `JobTitleDropdown` around? Seems unlikely to be useful.

File: frontend/src/pages/Admin/UserInfo.js (hunk 1 above)
Comment (kryswisnaskas) [reply frontend/src/pages/Admin/UserInfo.js:9]: Ah yes, I meant to remove it, but then it never happened. Thanks for catching it.

**Review by rahearn — APPROVED** [review]

**Pull request merged**